.github/workflows/pr_review.yml

```diff
@@ -30,8 +30,8 @@ jobs:
           OPENAI_API_KEY: ${{ secrets.OPENAI_API_KEY }}
         with:
           debug: true
-          review_simple_changes: false
-          review_comment_lgtm: false
+#          review_simple_changes: false
+#          review_comment_lgtm: false
```

Comment:
The line is currently commented out. Make sure to remove the '#' character to uncomment it.